src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -81,7 +73,31 @@ protected void runTool(final JavaSparkContext ctx) {
             }
 
         });
-        assembledBreakpoints.saveAsTextFile(output);
+        allContigAlignments.saveAsTextFile(output);
+    }
+
+    /**
+     * Loads an RDD of {@link ContigsCollection} objects keyed by assembly ID from disk. The input file
+     * should be the output of as RunSGAViaProcessBuilderOnSpark.
+     */
+    static JavaPairRDD<String, ContigsCollection> loadContigsCollectionKeyedByAssemblyId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
+
+        final JavaPairRDD<String, String> contigCollectionByBreakpointId =
+                inputAssemblies
+                        .flatMapToPair(RunSGAViaProcessBuilderOnSpark::splitAssemblyLine);
```

Comment:
What I've typically done in the past is to move a nested class to an independent, top-level class as soon as two programs need it.  So, by that rule, ContigsCollection should be in its own file.  And it could have some static methods for handling I/O.
I've never been completely happy about cluttering up a class like ContigsCollection with knowledge about I/O and external formats, but maybe that's the least evil.  I'd been putting that stuff in SVUtils, but I'm not really happy with that idea, either.  Maybe the right way to do it is to have another top-level class, ContigsCollectionIO.  Maybe we can chat about it and decide on a style, and stick with that.